index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Diego Vidal</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet"> -->
+
+</head>
+
+<body>
+    <div class="body-wrap">
+
+        <div class="nav-flex-container">
+            <div class="nav-logo">My Logo</div>
+            <button class="nav-menu-button"><img class=" nav-logo" src="./icons/Icon - Menunav-button.png"
+                    alt="navegation button"></button>
+            <p>Portfolio</p>
+            <p>About</p>
+            <p>Contact</p>
+        </div>
+
+        <header class="header-container">
+
+            <section class="header-text">
+                <h1>
+                    I’m Alexandra. <br>
+                    Glad to see you!
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and
+                    experience! If you like what you see and have a project
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="lets-conect">
```

Comment:
To make your design line up with the figma design, consider fixing the font weight for the `h1` and the font size and weight for the paragraph


Your design

<img width="209" alt="weight" src="https://user-images.githubusercontent.com/84930355/235006847-4d00885f-aa3f-4a11-9890-bea8c4f2567f.PNG">



Figma
<img width="258" alt="qwe" src="https://user-images.githubusercontent.com/84930355/235006772-014ed06b-e97e-4b26-bc13-7f39899348c3.PNG">

Reply:
I think you are right here, but I'm facing some difficulties here, I have been trying to make letters bold to match the requirements but for some reason, the "font-weight" property is not affecting my text elements if you could give me more insight would appreciate it!"
I tried: 
``
h1 {
font-weight: bold;
font-weight: 900;
}
``
one more time thanks for your time @DuaneDave !